tests/settings.py

```diff
@@ -54,14 +55,18 @@
 SECRET_KEY = 'verysecret'
 
 # weaker password hashing shoulod allow for faster tests
-PASSWORD_HASHERS = (
+PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.CryptPasswordHasher',
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
     'django.contrib.auth.hashers.BCryptPasswordHasher',
     'django.contrib.auth.hashers.SHA1PasswordHasher',
     'django.contrib.auth.hashers.MD5PasswordHasher',
-)
+]
+# Django 5.0 removed this hasher
+if DJANGO_VERSION >= DJANGO_42:
```

Comment:
Should this be `DJANGO_VERSION > DJANGO_42`, given that it wasn't removed until 5.0?

Reply:
Should be good now.